bootstrap-joining-demo/aws-api-ec2/README.md

```diff
@@ -4,6 +4,11 @@ Your Akka Cluster on Vanilla EC2 using Tag Based Discovery
 Note: these steps can be easily automated via CloudFormation or 
 Python / shell scripts etc. In fact, a simple CloudFormation script is provided [here](src/main/resources/CloudFormation).  
 
+Examine the source code
+-----------------------
+In `application.conf`, you'll notice the lack of need for manual configuration of IPs
+or host names for the cluster to form - the discovery mechanisms will do this for us.
+
```

Comment:
this introduction used to be there in the beginning but somehow got accidentally removed